utility/src/main/java/org/oppia/android/util/parser/image/UrlImageParser.kt

```diff
@@ -151,6 +152,8 @@ class UrlImageParser private constructor(
 
     override fun onResourceReady(resource: T, transition: Transition<in T>?) {
       val drawable = retrieveDrawable(resource)
+      // This statement is logged correctly
+      Log.d("TAGG", "onResourceReady: Top " + htmlContentTextView.text)
```

Comment:
@rt4914 added a comment here, this log statement is logged correctly and printing, alt-with-value text